tests/test_synced_collections/test_json_collection.py

```diff
@@ -15,6 +16,8 @@
     JSONList,
 )
 
+WINDOWS = sys.platform.startswith("win32") or sys.platform.startswith("cygin")
```

Comment:
Minor suggestion for a better variable name and fix a typo:
```suggestion
ON_WINDOWS = sys.platform.startswith("win32") or sys.platform.startswith("cygin")
```